lib/license.js

```diff
@@ -0,0 +1,76 @@
+// useful helpers for bootstrapping npmE install.
```

Comment:
Is there a way we can Dockerize this tool? My (very loose) plan is to have this repo on each production machine and alternate the config, so I'd rather not have javascript dependencies in this repo. The npm scripts are optional and only really helpful at dev time.

In addition, this seems useful as a separate module — could it be pulled in as a `npmo-license` module with a `bin` executable?